### DISCUSSION TITLE
Use blink main branch

### DISCUSSION THREAD

**Discussion by s1n7ax** [LazyVim/LazyVim#5180] [Q&A] [answered]
I saw this message somewhere that if I want to use the `main` of blink plugin enable some option or something. I couldn't find it anymore in release notes or in the coding.blink extra documentation. What should I do to use latest from blink?

**Comment by folke** [comment, marked as answer]
Make sure you have a working rust install, and then add the below to your `options.lua`:

```lua
vim.g.lazyvim_blink_main = true
```

**Reply by s1n7ax** [reply]
Thanks
